torchmultimodal/modules/layers/normalizations.py

```diff
@@ -45,3 +46,25 @@ def forward(self, x: Tensor) -> Tensor:
             self.eps,
         )
         return output.type_as(x)
+
+
+class RMSNorm(nn.Module):
+    """Root Mean Square layer normalization
+    as proposed in: https://arxiv.org/abs/1910.07467
+
+    params:
+    dim = model size
+    eps = epsilon
```

Comment:
nit: follow this format https://github.com/facebookresearch/multimodal/blob/main/torchmultimodal/models/dalle2/dalle2_decoder.py#L48 (type, default values etc)

Reply:
updated to use similar format